Sources/UberAuth/Authorize/AuthorizationCodeAuthProvider.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Copyright © Uber Technologies, Inc. All rights reserved.
+//
+
+import AuthenticationServices
+import Foundation
+import UberCore
+
+public final class AuthorizationCodeAuthProvider: AuthProviding {
+    
+    // MARK: Initializers
+    
+    public init() {}
+    
+    // MARK: Public Properties
+    
+    public typealias Completion = (Result<Client, UberAuthError>) -> Void
+    
+    // MARK: AutoProviding
+    
+    public func execute(authDestination: AuthDestination,
+                        prefill: Prefill?,
+                        completion: @escaping Completion) {
+        // TODO: Implement
```

Comment:
Should not have this class at all right now, considering that it is no-op at this point?

Reply:
This will be implemented in the next PR. I'm including it for testing purposes.